crates/cxx-qt-lib-headers/include/core/qcoreapplication.h

```diff
@@ -14,6 +14,7 @@
 #include <QtCore/QCoreApplication>
 #include <QtCore/QStringList>
 #include <QtCore/QVector>
+#include <QtGui/QFont>
```

Comment:
does this not need the if CXX_QT_GUI_FEATURE ?

Reply:
ah yep indeed.